CONTRIBUTING.md

```diff
@@ -164,6 +186,51 @@ After you get your container up and running, you can test it by adding test asse
 
 Finally, commit your changes and submit a PR - we'll take a look at it, provide any needed feedback, and then merge it in. We appreciate any and all feedback!
 
+### Adding a Database Definition to an existing Container
+
+VS Code Dev Containers allow multiple containers to run together. This is useful, for example, when adding database support to an existing container, and can be leveraged with `docker compose.`
+
+1. Create a new copy of an existing language definition folder, and hyphenate the database name.
+   1. For example, `java` becomes `java-postgres`.
+2. Create a new `docker-compose.yml` file.
+   1. Add the existing `Dockerfile` to the `docker-compose` as an `app`. This will be the Dev Container where your code gets executed.
+   2. Add the definition for the database to the `docker-compose`, version tagging it or using the `latest`. Most usage scenarios don't require custom images for the database.
+   3. It's common that containers will need to be connected through a network so that the `app` can connect to the `db`. This can normally be done by adding the following to the `app` definition:
+
+    ```yaml
+        network_mode: service:db
+    ```
+
+3. Modify the `devcontainer.json` with any launch options or VS Code specific `settings.json` options that may be needed. 
+   1. For example, when working with a SQL database, it might be necessary to modify the settings with something like the following:
+   ```json
+   		"sqltools.connections": [
```

Comment:
would it be worth poinging out that in this example, the settings are specific to the SQL Tools VS Code extension?